src/MovieCard/MovieCard.js

```diff
@@ -127,39 +115,29 @@ class MovieCard extends Component {
     }
     return radioButtons
   }
-
   createTomatoElement = () => {
-    const tomato = <img
+    const tomato = 
+			<img
           className="rating-img"
           src="https://cdn.iconscout.com/icon/premium/png-256-thumb/tomato-1640383-1391081.png"
           alt="Tomato"
-        />
-    const greenPaint = <img
+       />
+    const greenPaint = 
+      <img
           className="rating-img"
           src="https://i.pinimg.com/originals/58/e0/a9/58e0a9b572353c77bb1a4b3f802f4cb8.png"
           alt="Green Paint Splatter"
-        />
-    if(this.state.foundRating) {
-      return this.state.foundRating >= 5 ? tomato : greenPaint
-    } else {
-      return this.props.averageRating >= 5 ? tomato : greenPaint
-    }
+      />
+    if (this.state.foundRating) return this.state.foundRating >= 5 ? tomato : greenPaint;
+    return this.props.averageRating >= 5 ? tomato : greenPaint;
   }
-
-	render = () => {
-    const className = this.state.clicked ? 'rating-form-container' : 'rating-form-container hide';
-    const tomatoElement = this.createTomatoElement()
-    const radioButtons = this.createRadioButtons();
-    const favoritedElement = this.props.userID ? this.createFavoriteBtn() : null
-    const movieCardClass = this.props.userID ? 'movie-card' : 'movie-card shorter'
+	render = () => { 
     return (
       <section className="movie-card-container" id={this.props.id}>
-        <section className={className}>
+        <section className={this.state.clicked ? 'rating-form-container' : 'rating-form-container hide'}>
           <form className="rating-form">
-            <div className="exit-btn" onClick={this.displayRatingForm}>
-              x
-            </div>
-            <div className="inputs">{radioButtons}</div>
+            <div className="exit-btn" onClick={this.displayRatingForm}>x</div>
+            <div className="inputs">{this.ratingButtons}</div>
```

Comment:
calling back to the constructor's ratingButtons to run through the for loop creating these